tools/dust/spec.nim

```diff
@@ -0,0 +1,84 @@
+import std/strutils
+import std/sets
+
+import
+  compiler / modules / modulegraphs,
+  compiler / ast / [ lineinfos, renderer, ast ]
+
+import hashing
+
+const
+  unsunny* = {nkCharLit..nkUInt64Lit} +
+             {nkFloatLit..nkFloat128Lit} +
+             {nkStrLit..nkTripleStrLit} +
+             {nkSym, nkIdent}
+
```

Comment:
Good call, I added a todo item to audit dust code to use the latest nimskull bits.

I just went for getting it to compile rather than combing through it.